content/id/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -0,0 +1,639 @@
+
+- Jika kamu ingin menggunakan IPv6--baik jaringan _dual-stack_, ataupun jaringan _single-stack_ IPv6
+  --untuk klastermu, pastikan _plugin_ jaringan Pod
+  mendukung IPv6.  
+  Dukungan IPv6 telah ditambahkan pada CNI sejak [v0.6.0](https://github.com/containernetworking/cni/releases/tag/v0.6.0).
+
+{{< /caution >}}
+
+{{< note >}}
+Saat ini Calico adalah satu-satunya _plugin_ CNI yang dapat menerima uji e2e (_end-to-end_) oleh proyek kubeadm.
+Jika kamu menemukan isu terkait _plugin_ CNI kamu harus membuat tiket pada pelacak isu masing-masing _plugin_
+alih-alih pada pelacak isu kubeadm maupun kubernetes.
+{{< /note >}}
+
+Beberapa proyek eksternal menyediakan jaringan Pod Kubernetes menggunakan CNI, beberapa di antaranya juga
+mendukung [Network Policy](/docs/concepts/services-networking/networkpolicies/).
+
+Lihat daftar
+[_add-on_ jejaring dan _network policy_](https://kubernetes.io/docs/concepts/cluster-administration/addons/#networking-and-network-policy) yang tersedia.
+
+Kamu dapat memasang _add-on_ jaringan Pod dengan perintah berikut pada simpul
+_control-plane_ atau simpul yang memiliki kredensial kubeconfig:
+
+```bash
+kubectl apply -f <add-on.yaml>
+```
+
+Kamu hanya dapat memasang satu jaringan Pod per klaster.
+Di bawah ini kamu dapat menemukan instruksi instalasi untuk beberapa _plugin_ jaringan Pod yang populer:
+
+{{< tabs name="tabs-pod-install" >}}
+
+{{% tab name="Calico" %}}
+[Calico](https://docs.projectcalico.org/latest/introduction/) merupakan penyedia jejaring dan _network  policy_. Calico mendukung sekumpulan opsi jejaring yang fleksibel sehingga kamu dapat memilih opsi yang paling efisien untuk situasimu, termasuk jaringan _non-overlay_ dan _overlay_, dengan atau tanpa BGP. Calico menggunakan mesin yang sama untuk melaksanakan _network policy_ pada _host_, _Pod_, dan (jika menggunakan Istio & Envoy) aplikasi yang berada pada lapisan _service mesh_. Calico bekerja pada beberapa arsitektur, meliputi `amd64`, `arm64`, dan `ppc64le`.
+
+Secara bawaan, Calico menggunakan `192.168.0.0/16` sebagai CIDR jaringan Pod, namun hal ini dapat diatur pada berkas calico.yaml. Agar Calico dapat bekerja dengan benar, kamu perlu memberikan CIDR yang sama pada perintah `kubeadm init` menggunakan opsi `--pod-network-cidr=192.168.0.0/16` atau melalui konfigurasi kubeadm.
+
+```shell
+kubectl apply -f https://docs.projectcalico.org/v3.11/manifests/calico.yaml
+```
+
+{{% /tab %}}
+
+{{% tab name="Cilium" %}}
+Agar Cilium dapat bekerja dengan benar, kamu harus memberikan `--pod-network-cidr=10.217.0.0/16` pada `kubeadm init`.
+
+Untuk menyebarkan Cilium kamu hanya perlu menjalankan:
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/cilium/cilium/v1.6/install/kubernetes/quick-install.yaml
+```
+
+Ketika seluruh Pod Cilium sudah bertanda `READY`, kamu dapat mulai menggunakan klaster.
+
+```shell
+kubectl get pods -n kube-system --selector=k8s-app=cilium
+```
+Keluaran akan tampil seperti berikut:
```

Comment:
```suggestion
Keluarannya akan tampil seperti berikut:
```

Reply:
Resolved in https://github.com/kubernetes/website/pull/20945/commits/03745eaec90064d48ba7c612f74c7825115b8ffc